frigate/config.py

```diff
@@ -46,6 +46,9 @@ class DetectorConfig(FrigateBaseModel):
 
 class UIConfig(FrigateBaseModel):
     use_experimental: bool = Field(default=False, title="Experimental UI")
+    show_confirmation_prompts: bool = Field(
+        default=False, title="Show confirmation prompts"
```

Comment:
I was thinking True may be a better default since IMO start on the safer side and if users don't want then can disable. Curious on others' thoughts though.

Reply:
Makes sense. I've defaulted it to True. But I'm happy either way.